src/routes/Profile/Favorites/Favorites.tsx

```diff
@@ -154,7 +157,7 @@ const Favorites = () => {
           return (
             <OpportunityCard
               key={id}
-              to={`/${language}/${t('slugs.job-opportunity.index')}/${id}`}
+              to={`/${language}/${getTypeSlug(mahdollisuusTyyppi)}/${id}`}
```

Comment:
Noticed this bug by chance and decided to fix it although it wasn't part of the ticket. The card was trying to route to job opportunity page even if favorite type was `KOULUTUSMAHDOLLISUUS`